src/styles/card.css

```diff
@@ -0,0 +1,53 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  width: 200px;
+}
+
+.card__image {
+  background-image: url(./../images/imac.jpeg);
+  background-repeat: no-repeat;
+  background-size: 160px 134px;
+  height: 134px;
+  width: 160px;
+  margin: 0 auto 40px;
+}
+
+.card__name {
+  margin: 0 0 4px;
+}
+
+.card__code {
+  margin: 0 0 16px;
+}
+
+.card__rating {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  margin-bottom: 24px;
+}
+
+.card__review {
+  margin: 0;
+}
```

Comment:
Move these and other related code inside of main class means BLOCK code. you can write it like &__elementName here & represent your main class.

Reply:
Here card is the block, and I am styling all it's elements inside of the card block. 
I don't know the  &__elementName way.